sqlmesh/core/linter/rule.py

```diff
@@ -22,6 +22,20 @@ class RuleLocation(PydanticModel):
     start_line: t.Optional[int] = None
 
 
+class Position(PydanticModel):
+    """The position of a rule violation in a file, the position follows the LSP standard."""
```

Comment:
Why do these wrapper classes inherit from `PydanticModel` instead of making them data classes? I think the latter is more lightweight, so may be worth exploring if we don't care about serializing these objects.

Reply:
🤷